src/main/java/org/jenkinsci/plugins/scm_filter/GitHubPullRequestSkipTrait.java

```diff
@@ -97,7 +97,7 @@ public boolean isExcluded (
                 int prNumber = Integer.parseInt(scmHead.getName().substring(3));
 
                 String prTitle = repository.getPullRequest(prNumber).getTitle();
-                return prTitle.matches("(?i).*\\[(wip|ci[ -_]skip|skip[ -_]ci)\\].*");
+                return prTitle.matches("(?i).*\\[(wip|ci[- _]skip|skip[- _]ci)\\].*");
```

Comment:
hmm isn't the solution to escape the special character?
`[ _\-]`

Reply:
Both solutions work.

Reply:
Yup, I know. You didn't see my point though. If I updated the regex to support additional character and naively added it before the `-` character, then the bug reappears. Escaping the `-` character is the more appropriate solution. If you don't want to change your implementation, I suggest you put a caveat comment.

Reply:
Please also include the reference to this bug. I'm not sure why we need to make this change, something else must not be working as intended as this breaks how regex works.

Reply:
Actually, I agree with @FerrielMelarpis 's comment that it should be escaped since we're using it as actual dash character, not a range.

Reply:
Done!